socorro/unittest/middleware/test_middleware_app.py

```diff
@@ -589,8 +596,17 @@ def put(self, server, url, data=None, expect_errors=False):
 
     def test_overriding_implementation_class(self):
         config_manager = self._setup_config_manager({
-            'implementations.service_overrides': 'Backfill: psql, Bugs: typo'
-        })
+            'implementations.implementation_list': (
+                'psql:socorro.unittest.middleware'
+            ),
+            'implementations.service_overrides': (
+                'Bugs: typo'
+            )
+        },
+            services_list=(
+                ('/bugs/', 'bugs.Bugs'),
+            )
```

Comment:
First of all, I barely remember the reasons for the changes. It's been so long. 

First of all; remember that middleware is dying. The Bugs PG service is the first of many PG resources I want to extract. Once that's done it'll be easy to do all the others. So don't worry about messing up `middleware_app.py`. It's a dying thing. 

Actually, in the midst of trying to respond I reviewed the code and I think you're right. It's nuts how much these tests depend on real classes to exist. So I wrote a new commit where it relies entirely on importing (and not being able to import) `socorro/unittest/middleware/footing.py` and `socorro/unittest/middleware/somesubmodule/footing.py`. Please check it out. No more mention of "bugs" here in this context.